libs/application/templates/operating-license/src/lib/OperatingLicenseTemplate.ts

```diff
@@ -59,6 +61,13 @@ const OperatingLicenseTemplate: ApplicationTemplate<
               ],
               write: 'all',
               delete: true,
+              api: [
+                PaymentCatalogApi.configure({
+                  params: { orginizationId: '6509142520' },
```

Comment:
`libs/application/templates/passport/src/lib/PassportTemplate.ts` stores the organization id in constants where it is easier to find and change.
Consider doing the same here?